examples/pumi/ex1.cpp

```diff
@@ -0,0 +1,247 @@
+//                                MFEM Example 1
```

Comment:
@KazemKamran: can you unify the example headers to be of this form, see e.g. the files inexamples/sundials and examples/petsc?

Reply:
Done